docs/Backup_and_restore_FARE.md

```diff
@@ -0,0 +1,115 @@
+# Backup and restore FARE
+
+The backup is composed of three parts:
+- Databse where are stored all the metadata and the users
+- Elasticsearch which is used to index the records and search them
+- Files which should be stored in a shared volume
+
+## 0. Shared volumes
+
+Data are shared between containers and host, they was added in the `docker-compose.full.yml` file and are organazied as follow:
+- To share the dump of the metadata between the `fare-invenio_db_1` container and the host, it was added  `/fare_shared_data/fare_db:/backup_fare_db` as volume in the `db` section

```

Comment:
Please always use absolute URLs when dealing with Linux + Docker

Reply:
ok, i'll check and fix all the paths